manifest_staging/charts/gatekeeper/crds/assign-customresourcedefinition.yaml

```diff
@@ -0,0 +1,185 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Are we okay with CRDs getting installed even if mutation is disabled?

Reply:
I tried to make this conditional on the field, but in helmv3 they removed the ability to use templates for CRDs. https://github.com/helm/helm/issues/7083